src/shared/components/results/ResultsButtons.tsx

```diff
@@ -121,7 +126,8 @@ const ResultsButtons: FC<ResultsButtonsProps> = ({
             })}
           />
         </Button>
-        {viewMode === ViewMode.TABLE && <CustomiseButton />}
+        {(viewMode === ViewMode.TABLE || disableCardToggle) &&
+          namespace !== Namespace.idmapping && <CustomiseButton />}
```

Comment:
As we are already using props to change what is displayed we could could provide a `disableCustomise` and keep exceptional Namespace behaviour in a parent component eg `IDMappingResult`